app/views/devise/registrations/new.html.erb

```diff
@@ -1,29 +1,34 @@
-<h2>Sign up</h2>
+<div class="modal-dialog modal-content p-4">
+
+  <h2>Sign up</h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+
+  <div class="form-group">
+    <%= f.label :email %>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", :required => true %>
   </div>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :password %>
     <% if @minimum_password_length %>
     <em>(<%= @minimum_password_length %> characters minimum)</em>
-    <% end %><br />
-    <%= f.password_field :password, autocomplete: "new-password" %>
+    <% end %>
+    <%= f.password_field :password, autocomplete: "new-password", class: "form-control", :required => true, :minlength => 6 %>
   </div>
 
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  <div class="form-group">
+    <%= f.label :password_confirmation %>
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control", :required => true, :minlength => 6 %>
```

Comment:
We can use avoid old hash rocket syntax, same for other fines also.
```suggestion
    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control", required: true, minlength: 6 %>
```

Reply:
Fixed in [`9a7d980` (#31)](https://github.com/suho/google-search-ruby/pull/31/commits/9a7d9809a136ebbf8cd9847aa9030614cfe2905c)